ci-operator/step-registry/telcov10n/functional/compute-nrop/e2e-testing/telcov10n-functional-compute-nrop-e2e-testing-commands.sh

```
@@ -0,0 +1,105 @@
+echo "Copy must gather to artifacts directory"
+
+scp -r -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /tmp/temp_ssh_key \
+  "${BASTION_USER}@${BASTION_IP}":/tmp/wip/artifacts/* "${ARTIFACT_DIR}"
+
+
+scp -r -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /tmp/temp_ssh_key \
+  "${BASTION_USER}@${BASTION_IP}":/tmp/wip/tests "${ARTIFACT_DIR}"
+
+
+echo "Copy junit test reports to shared directory for reporter step"
+if ls ${ARTIFACT_DIR}/tests/junit/*.xml 1> /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)